core-ui/src/shared/constants.js

```diff
@@ -1,9 +1,9 @@
-export const PREFERENCES_TITLE = 'Preferences';
-export const CLUSTERS_OVERVIEW_TITLE = 'Clusters Overview';
-export const ADD_CLUSTER_TITLE = 'Add Cluster';
-export const API_RULES_TITLE = 'API Rules';
-export const NO_PERMISSIONS_TITLE = 'No Permissions';
-export const CLUSTER_OVERVIEW_TITLE = 'Cluster Overview';
```

Comment:
Since we're using the translations everywhere, I think we should just move those constants inline.